.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,23 @@
-| Required Info  |   |
-|---|---|
-| Camera Model | R200 / F200 / SR300 / ZR300 / D400 | 
-| Firmware Version |   | 
-| Operating System & Version |   |
-| Kernel Version (Linux Only)  |   |
-| SDK Version | |
+* Before opening a new Issue please first try the following (Click "Preview" above for a better view):
 
+    * This SDK is **not** the [Intel RealSense SDK for Windows](https://software.intel.com/en-us/realsense-sdk-windows-eol) which was discontinued. For issue with that, please [contact support](https://www.intel.com/content/www/us/en/support/contact-support.html).
+    * If you are using R200, F200 or ZR300, please note that the support for these devices is maintained only via the [legacy branch](https://github.com/IntelRealSense/librealsense/tree/legacy).
+    * Have you tried following our [examples](https://github.com/IntelRealSense/librealsense/tree/master/examples#sample-code-for-intel-realsense-cameras)?
```

Comment:
Please rephrase in more encouraging way. Anyone is welcomed to open an issue regardless of if he / she done any of the recommended tasks. 
"Consider checking out SDK examples" or something like this.